custom/icds/translations/integrations/client.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+import requests
+import os
+import json
+
+
+class TransifexApiClient():
+    def __init__(self, token, orgranization, project):
+        self.username = "api"
+        self.token = token
+        self.organization = orgranization
+        self.project = project
+
+    @property
+    def _auth(self):
+        return (self.username, self.token)
+
+    def list_resources(self):
+        url = "http://api.transifex.com/organizations/{}/projects/{}/resources".format(
+            self.organization,
+            self.project
+        )
+        return requests.get(url, auth=self._auth)
+
+    def delete_resource(self, resource):
+        url = "https://www.transifex.com/api/2/project/{}/resource/{}".format(
+            self.project, resource)
+        return requests.delete(url, auth=self._auth)
+
+    def upload_resource(self, path_to_pofile, resource_slug, resource_name):
+        url = "https://www.transifex.com/api/2/project/{}/resources".format(self.project)
+        content = open(path_to_pofile, 'r').read()
```

Comment:
ya, i don't think it will retain keep any reference to the file.

> is the pofile kept as a temporary file on the filesystem?
No its not. Its generated in the current directory and then deleted at the end of the process here itself. I would like us to remove them here because they can be many in number and in case this rolls out as a feature for multiple domains and is used, it would result in quite some files which would stay around till they are removed automatically.
Or can also make them temp file but still remove them if that sounds better, though not sure why, possibly in case this process fails and does not reach the step to remove them manually?